style/modules/grid.css

```diff
@@ -0,0 +1,20 @@
+.gridContainer{ /* a container with grid */
```

Comment:
also, let's call this file "gridContainer.css" (or rename the module to `.grid` -- I'm fine with either, but they should match)